src/cli/project.py

```diff
@@ -232,55 +291,41 @@ def up(ctx, project, organization, ingress, provider, workers, **kwargs):
     Start/Resume a project cluster.
     """
 
-    # GraphQL
     try:
         graph_ql = GraphQL(authentication=ctx.auth)
-        data = graph_ql.query(
-            """
-            {
-                allProjects {
-                    results {
-                        id
-                        title
-                        clusterSettings {
+        if not organization:
```

Comment:
I think this is over complicated. This query could use the `organizationId` filter from "allProjects"

![Screenshot from 2021-07-30 13-09-31](https://user-images.githubusercontent.com/6874607/127646905-849acbbc-9588-4e2e-8fb7-ef43c9bcc9a4.png)